setup.py

```diff
@@ -15,7 +15,7 @@
     packages=find_packages('src/'),
     package_dir={'': 'src'},
     install_requires=[
-        "pyop >= 3.0.1",
+        "pyop >= 3.0.1", # TODO
```

Comment:
```suggestion
        "pyop >= 3.2.0",
```

pyop now needs redis or mongo from extra requires. We can bubble up this choice here, too:

```
    extras_require={
        "ldap": ["ldap3"],
        "redis": ["pyop[redis]"],
        "mongo": ["pyop[mongo]"],
    },
```

We should now be able to do `pip install satosa[redis]` and get the correct dependencies.

_(needs to be tested)_

Reply:
This is fixed by #375